themes/theme-ios11/pages/index.jsx

```diff
@@ -56,6 +57,7 @@ const Pages = ({ store }) => (
               <Route pattern={ITEM_WRITE_REVIEW_PATTERN} component={routes.WriteReview} />
               <Route pattern={CART_PATH} component={routes.Cart} />
               <Route pattern={BROWSE_PATH} component={routes.Browse} preload />
+              <Route pattern={MORE_PATH} component={routes.More} preload />
```

Comment:
Consider not using `preload`. The more page looks the same anyway and might improve performance.

Reply:
I've removed it